contracts/staking/StakingPool.sol

```diff
@@ -486,6 +497,10 @@ contract StakingPool is
         return block.timestamp >= _rewardsAvailableTimestamp;
     }
 
+    function _isMinTotalPoolStakeMet() internal view returns (bool) {
```

Comment:
question: why the `internal` modifier over the `private`?

_This changeset contains only edits to this one Solidity file_

Reply:
Yeah it was just because the other "conditional" checks were also `internal`. I've changed it to private - but perhaps we should create an issue to change the other functions `_isStakingPeriodComplete` , `_isRewardsAvailable` etc to private

Reply:
Yes, please create that issue 👍

Reply:
Created, add details as appropriate https://github.com/windranger-io/windranger-treasury/issues/375